### ISSUE TITLE
I could not find function class_weights() in the utils.py

### ISSUE THREAD

**Issue by monkeykim111** [AnjanDutta/sem-pcyc#29]
I am trying to train a model with images and sketches with my data set.
So I used train_image.py and sketch_image.py in the utils folder.

**However, I get an error that there is no class_weights function used to find weights in line 116.**
![image](https://user-images.githubusercontent.com/82593754/142554357-7119df2d-5723-47c1-9e7b-889c2cd314b7.png)

![image](https://user-images.githubusercontent.com/82593754/142553966-384bd7d8-5e97-4d9c-a1ad-9b98833a74a0.png)

The class_weights() function could not be found among various functions defined in utils.py.
**1. Where is the class_weight function?
If not, how do I define it?**
or
**Is class_weights() correct for sklearn.utils.class_weight.compute_class_weight()?**

And there is one more issue.
**2. Where is the utils.accuracy() function? 
Without this too, how can we define it?**

![image](https://user-images.githubusercontent.com/82593754/142580658-c649f2e3-ae98-4a16-8969-9ecfb036bc46.png)
I checked that the train epoch was running, but I couldn't check the accuracy.

please answer about my question.........
Thank you

**Comment by AnjanDutta** [comment]
The `class_weights` and `accuracy` functions look like as below. I hope this helps.

```
def class_weights(clss):
    uniq_clss, clss_cnts = np.unique(clss, return_counts=True)
    wts = np.zeros(len(clss))
    for i in range(len(clss)):
        wts[i] = clss_cnts[np.where(uniq_clss == clss[i])]
    wts = 1 / wts
    return wts
```

```
def accuracy(output, target, topk=(1,)):
    """Computes the precision@k for the specified values of k"""
    maxk = max(topk)
    batch_size = target.size(0)
    _, pred = output.topk(maxk, 1, True, True)
    pred = pred.t()
    correct = pred.eq(target.view(1, -1).expand_as(pred))
    res = []
    for k in topk:
        correct_k = correct[:k].view(-1).float().sum(0)
        res.append(correct_k.mul_(100.0 / batch_size))
    return res
```
